WebService/WebService.csproj

```diff
@@ -36,12 +36,12 @@
         </Content>
     </ItemGroup>
     <ItemGroup>
-        <PackageReference Include="Autofac" Version="4.9.4" />
-        <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="5.0.1" />
+        <PackageReference Include="Autofac" Version="5.1.2" />
+        <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="6.0.0" />
```

Comment:
How did you test these changes, since it is a major version change (and potentially a breaking change)?